### PR TITLE
Unified component: Info tab within Venues (in alignment with Tables) 

### PR DESCRIPTION
**Pull request by dmitrii-zolotukhin**
- [x] Unify component Info Tab
- [x] Display component on all venues
- [x] Implement new design for Info Tab, check that "What's On" button doesn't close Info Tab

closes for sparkletown/internal-sparkle-issues#1070

### CODE REVIEW

File: src/components/organisms/InfoTab/InfoTab.scss [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+$InfoTab-position-top: 188px;
+$InfoTab-width: 70px;
+$InfoTab-width-expanded: 464px;
+$InfoTab-body-margin-top: 56px;
+
+$InfoTab-logo-width: 50px;
+$InfoTab-logo-height: $InfoTab-logo-width;
+$InfoTab-logo-width-expanded: 124px;
+$InfoTab-logo-height-expanded: $InfoTab-logo-width-expanded;
```
Comment (goran-peoski-work) [inline src/components/organisms/InfoTab/InfoTab.scss:11]: Following BEM convention when most if not all the rest of the `$scss-constants` start with a lower case.

File: src/components/organisms/InfoTab/InfoTabLogo.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+type ValidLogoIconName = "ambulance" | "create" | "heart" | "info";
+
+const logoMap = new Map<ValidLogoIconName, IconDefinition>([
+  ["ambulance", faAmbulance],
+  ["create", faEdit],
+  ["heart", faHeart],
+  ["info", faInfoCircle],
+]);
```
Comment (goran-peoski-work) [inline src/components/organisms/InfoTab/InfoTabLogo.tsx:24]:
```suggestion
const logoMap: Record<ValidLogoIconName, IconDefinition> =  {
  "ambulance": faAmbulance,
  "create": faEdit,
  "heart": faHeart,
  "info": faInfoCircle,
};
```

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
Some idiosyncratic code style changes could use retouch and a different approach to event bubbling up and CSS selector precedence

File: src/components/molecules/UserList/UserList.tsx [hunk 3]
```diff
@@ -62,20 +60,36 @@ export const UserList: React.FC<UserListProps> = ({
...
+  );
+
+  const onClickShowMore = useCallback(
+    (e: React.MouseEvent) => {
+      e.stopPropagation();
+      toggleExpanded();
+    },
+    [toggleExpanded]
+  );
+
```
[the remark this answers is not in the document]
Comment (dmitrii-zolotukhin) [reply src/components/molecules/UserList/UserList.tsx]: @goran-peoski-work fixed in 859a215d7eb1c77f050fef74791fc4b872dda80a

File: src/components/atoms/UserAvatar/UserAvatar.tsx [hunk 4]
```diff
@@ -22,7 +22,7 @@ export interface UserAvatarProps extends ContainerClassName {
   imageClassName?: string;
   showNametag?: UsernameVisibility;
   showStatus?: boolean;
-  onClick?: () => void;
+  onClick?: (e: React.MouseEvent) => void;
```
[the remark this answers is not in the document]
Comment (dmitrii-zolotukhin) [reply src/components/atoms/UserAvatar/UserAvatar.tsx]: @goran-peoski-work fixed in 859a215

File: src/components/organisms/InfoTab/InfoTab.scss [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+    border-radius: $border-radius--max;
+    object-fit: cover;
+
+    &--expanded {
+      width: $InfoTab-logo-width-expanded !important;
+      height: $InfoTab-logo-height-expanded !important;
+      border-radius: $border-radius--lg;
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (dmitrii-zolotukhin) [reply src/components/organisms/InfoTab/InfoTab.scss]: @goran-peoski-work fixed in 859a215

File: src/components/organisms/InfoTab/InfoTab.scss (hunk 1 above)
Comment (dmitrii-zolotukhin) [reply src/components/organisms/InfoTab/InfoTab.scss]: @goran-peoski-work fixed in 859a215

File: src/components/organisms/InfoTab/InfoTabLogo.tsx (hunk 2 above)
Comment (dmitrii-zolotukhin) [reply src/components/organisms/InfoTab/InfoTabLogo.tsx]: @goran-peoski-work fixed in 859a215

**Comment by sofisparkle** [comment]
@dmitrii-zolotukhin this should be abandoned for now as we have integrated it into schedule

**Pull request closed without merging**